domain/domain.go

```diff
@@ -1729,6 +1730,21 @@ func (do *Domain) MockInfoCacheAndLoadInfoSchema(is infoschema.InfoSchema) {
 	do.infoCache.Insert(is, 0)
 }
 
+func (do *Domain) renewLease() {
+	defer func() {
+		do.wg.Done()
+		logutil.BgLogger().Info("renew lease goroutine exited.")
+	}()
+	for {
+		select {
+		case <-do.exit:
+			return
+		case op := <-do.renewLeaseCh:
```

Comment:
When the `do.exit` receive message, and the goroutine exit,
someone may still sending message to  `do.renewLeaseCh`  and block forever...
It's a bug, but for domain, this is usually not a problem because the process is about to quit soon.